src/ImageSharp/Formats/Jpeg/Common/ZigZag.cs

```diff
@@ -11,34 +12,61 @@ namespace SixLabors.ImageSharp.Formats.Jpeg.Common
     /// unzig[3] is the column and row of the fourth element in zigzag order. The
     /// value is 16, which means first column (16%8 == 0) and third row (16/8 == 2).
     /// </summary>
+    [StructLayout(LayoutKind.Sequential)]
     internal unsafe struct ZigZag
     {
         /// <summary>
         /// Copy of <see cref="Unzig"/> in a value type
         /// </summary>
-        public fixed int Data[64];
+        public fixed byte Data[64];
```

Comment:
Good question if it's worth to use byte buffer here.

Doesn't worth to bother with it but a few notes for future ourselves:
- Indexing `byte[]` is most likely slower, but the buffer itself is smaller, increasing the chance to fit the CPU cachelines.
- Would be nice to benchmark the micro-scenario

Reply:
I thought it would be better cache wise. Benchmark is looking good with it also